Protocols/ViewController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ViewController.swift
+//  Protocols
+//
+//  Created by Felipe Gil on 2021-09-24.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    var deckOfCards = DeckofCards()
```

Comment:
Considering this is user only in the scope of `viewDidLoad` there is not reason to keep it hanging around after that method is done.

Reply:
Not sure how to solve this.